src/libraries/Common/src/Interop/Unix/System.IO.Ports.Native/Interop.Termios.cs

```diff
@@ -48,7 +48,7 @@ internal enum Queue
         internal static extern int TermiosGetSpeed(SafeSerialDeviceHandle handle);
 
         [DllImport(Libraries.IOPortsNative, EntryPoint = "SystemIoPortsNative_TermiosAvailableBytes", SetLastError = true)]
-        internal static extern int TermiosGetAvailableBytes(SafeSerialDeviceHandle handle, Queue input);
+        internal static extern int TermiosGetAvailableBytes(SafeSerialDeviceHandle handle, bool fromReadBuffer);
```

Comment:
```suggestion
        internal static extern int TermiosGetAvailableBytes(SafeSerialDeviceHandle handle, int fromReadBuffer);
```

since its prototype in pal_termios.h has `int32_t` type.

Reply:
You're right, but I prefer to use bool because it's reflect better the usage.
I will add marshaling to correct it.

```suggestion
        internal static extern int TermiosGetAvailableBytes(SafeSerialDeviceHandle handle, [MarshalAs(UnmanagedType.I4)]bool fromReadBuffer);
```

Reply:
You're right, but I prefer to use bool because it's reflect better the usage.
I will add marshaling to correct it.

```suggestion
        internal static extern int TermiosGetAvailableBytes(SafeSerialDeviceHandle handle, [MarshalAs(UnmanagedType.I4)]bool fromReadBuffer);
```